contracts/fate.md

```diff
@@ -919,6 +919,90 @@ the RLP encoding of the RLP encoding of the empty map i.e. the bytes 130, 47, 0.
 
 An empty FATE contract is encoded as the byte sequence 128, 130, 47, 0, 130, 47, 0.
 
+# Appendix 2: Delegation signatures
+
+A couple of FATE operations can perform actions on behalf of a "user", these
+operations are: `AENS_PRECLAIM`, `AENS_CLAIM`, `AENS_UPDATE`, `AENS_TRANSFER`,
+`AENS_REVOKE`, `ORACLE_REGISTER`, `ORACLE_RESPOND`, and `ORACLE_EXTEND`. The
+first argument of these operations is a _delegation signature_ - the signature
+"proves" that the user is allowed to manipulate the object (name or oracle).
+Naturally, if the contract itself is the owner of the name/oracle the signature
+is not needed. Otherwise, the user signs (using the private key) data
+authorizing the operation. Note: it is not possible to make a delegation
+signature using a generalized account.
+
+There are five different delegation signatures:
+ - `AENS_PRECLAIM` - the user signs: `owner account + contract`
+ - `AENS_CLAIM`,`AENS_UPDATE`, `AENS_TRANSFER`, `AENS_REVOKE` - the user signs:
+   `owner account + name hash + contract`
+ - AENS wildcard (valid for any name) - the user signs:
+   `owner account + contract` [New in `FATE_03`]
+ - `ORACLE_REGISTER`, `ORACLE_EXTEND` - the user signs: `owner account + contract`
+ - `ORACLE_RESPOND` - the user signs: `query id + contract`
+
+To protect about cross network re-use of signatures, the data to be signed is
+also prefixed with the _network id_.
+
+## From Ceres: Serialized signature data
+
+From Ceres/`FATE_03` the material to be signed is more structured; the reason
+is two-fold, (a) to make it easier for the signer to see the what is signed and
+why, and (b) to safeguard against spoofing a transaction as a delegation
+signature (their structure was similar). Note: the semantic data to be signed
+has not changed from the description above, only the exact bytes to be signed
+and its structure.
+
+For general information about serialization, RLP encoding, etc, see the general
+[serialization document](../serializations.md).
+
+We use the tag `0x1a01` to identify delegation signatures. We use the following
+tags/types to identify the different delegation signatures:
+
+| Tag | Delegation type |
```

Comment:
With this tag, we don't need prefixed ids and can use binary as before, but I don't have a preference.

Reply:
Let's do it like this, it is in line with the other serializations...